doc/apiary/orioncontextbroker.apib

```diff
@@ -224,9 +223,9 @@ And the response will be;
 </center>
 + Request (application/json)
 
-    + Headers (application/json)
+    + Headers 
     
-        Accept: application/json
+                Accept: application/json
```

Comment:
It seems that the FABRE tool that process the .apib file needs such indentation.

NTC